lib/galaxy/jobs/runners/__init__.py

```diff
@@ -378,6 +379,19 @@ def _walk_dataset_outputs(self, job: model.Job):
         #      yield (dataset_assoc, dataset_assoc.dataset)
         #  I don't understand the reworking it backwards.  -John
 
+    def _verify_celery_config(self):
+        if not self.app.config.enable_celery_tasks:
+            raise ConfigurationError("Can't request celery metadata without enabling celery tasks")
+        celery_conf = self.app.config.celery_conf
+        if not celery_conf:
+            raise ConfigurationError(
+                "Celery backend not set. Please set `result_backend` on the `celery_conf` config option."
+            )
+        else:
+            result_backend = celery_conf.get("result_backend", None)
+            if result_backend != "rpc://localhost":
+                raise ConfigurationError(f"Wrong celery backend: {result_backend}")
```

Comment:
I ... think that as long as there is any backend this is fine, we can drop this, that was for debugging tests.